app/javascript/styles/components.scss

```diff
@@ -3798,6 +3806,182 @@ button.icon-button.active i.fa-retweet {
   z-index: 5;
 }
 
+.video-player {
+  overflow: hidden;
+  position: relative;
+  background: $base-shadow-color;
+
+  &:-webkit-full-screen {
+    width: 100%;
+    height: 100%;
```

Comment:
`!important` is needed to override inline styles. Firefox has those styles in UA stylesheet.

also this can be merged into `&.fullscreen`?

Reply:
and add `margin: 0` to override `margin-top` on `.status .video-player`.